app/Models/Traits/UsesLinkedFields.php

```diff
@@ -11,18 +11,22 @@ public function updateAllAnswers(array $input): array
         $localAnswers = [];
         foreach ($this->form->sections as $section) {
             foreach ($section->questions as $question) {
-                $value = data_get($input, $question->uuid);
-                if ($value) {
-                    if (empty($question->linked_field_key)) {
-                        $localAnswers[$question->uuid] = data_get($input, $question->uuid);
-                    }
-                    $linkedFieldInfo = $question->getLinkedFieldInfo([
-                        'organisation' => $this->organisation,
-                        'project-pitch' => $this->projectPitch,
-                    ]);
+                if ($question->input_type === 'conditional') {
+                    $localAnswers[$question->uuid] = data_get($input, $question->uuid);
+                } else {
+                    $value = data_get($input, $question->uuid);
+                    if ($value) {
+                        if (empty($question->linked_field_key)) {
+                            $localAnswers[$question->uuid] = data_get($input, $question->uuid);
+                        }
+                        $linkedFieldInfo = $question->getLinkedFieldInfo([
+                            'organisation' => $this->organisation,
+                            'project-pitch' => $this->projectPitch,
+                        ]);
 
-                    if (! empty($linkedFieldInfo)) {
-                        $this->updateLinkedFieldValue($linkedFieldInfo, data_get($input, $question->uuid));
+                        if (! empty($linkedFieldInfo)) {
+                            $this->updateLinkedFieldValue($linkedFieldInfo, data_get($input, $question->uuid));
+                        }
                     }
```

Comment:
I think the code handles too many conditional and we can apply "early return" or "early exit" pattern.

Meaning that instead of handling cases in this way:
```
if (condition) {
//something
} else {
//something else 
}
```

We can apply something like 
```
if (!condition) {
//something else
   continue;
 }
 //something
 ```